### PR TITLE
Jest

### PR DESCRIPTION
**Pull request by samblackk**
Attached to https://github.com/astronomer/astronomer/issues/357

Jest unit testing for Orbit. 
Open for collaboration - help write some tests!
Run `npm run test:watch` to hot-build tests. 

Coverage is set to 90%

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -142,6 +142,37 @@
...
+      "/src/instruments/api/",
+      "/src/instruments/context/",
+      "/src/instruments/forms/",
+      "/src/instruments/react/"
+    ],
+    "testPathIgnorePatterns": [
+      "/tools/",
+      "/src/helpers/",
+      "/src/modules/",
+      "/src/instruments/analytics/",
```
Comment (samblackk) [inline package.json]: This will be updated. For now - writing tests for each "large" section in order to chuck the workload.

**Comment by ashb** [comment]
❤️  tests!

**Comment by samblackk** [comment]
@schnie Let's merge it! Coverage is set to 90% and we're at 100% for the current components included in the test suites. CI runs through them - outputting verbose info per test and will bail if either coverage is lower than the threshold OR if a test fails.

**Comment by ashb** [comment]
Love it @samblackk. Anything we need to know about the snapshots? When I should update them etc?

**Pull request merged**